packages.csv

```diff
@@ -0,0 +1,26 @@
+# Do not remove these comments
+# Format of the file : <Distro Name>,<Fuse Version, Architecture, File Type>,<Repo Name>,<Release Name(for deb repos only)>
+Ubuntu-18.04,azcopyAmdDeb,microsoft-ubuntu-bionic-prod-apt,bionic
+Ubuntu-20.04,azcopyAmdDeb,microsoft-ubuntu-focal-prod-apt,focal
+Ubuntu-22.04,azcopyAmdDeb,microsoft-ubuntu-jammy-prod-apt,jammy
+Ubuntu-22.04,azcopyArmDeb,microsoft-ubuntu-jammy-prod-apt,jammy
+Ubuntu-24.04,azcopyAmdDeb,microsoft-ubuntu-noble-prod-apt,noble
+Ubuntu-24.04,azcopyArmDeb,microsoft-ubuntu-noble-prod-apt,noble
+Debian-9.0,azcopyAmdDeb,microsoft-debian-stretch-prod-apt,stretch
+Debian-10.0,azcopyAmdDeb,microsoft-debian-buster-prod-apt,buster
+Debian-11.0,fuse3AmdDeb,microsoft-debian-bullseye-prod-apt,bullseye
+RHEL-7.5,azcopyAmdRpm,microsoft-rhel7.5-prod-yum,
+RHEL-7.8,azcopyAmdRpm,microsoft-rhel7.8-prod-yum,
+RHEL-8.1,azcopyAmdRpm,microsoft-rhel8.1-prod-yum,
+RHEL-8.2,azcopyAmdRpm,microsoft-rhel8.2-prod-yum,
+RHEL-7.0,azcopyAmdRpm,microsoft-rhel7.0-prod-yum,
+RHEL-8.0,azcopyAmdRpm,microsoft-rhel8.0-prod-yum,
+RHEL-9.0,azcopyAmdRpm,microsoft-rhel9.0-prod-yum,
+RHEL-9.0,azcopyArmRpm,microsoft-rhel9.0-prod-yum,
+CentOS-7.0,azcopyAmdRpm,microsoft-centos7-prod-yum,
+CentOS-8.0,azcopyAmdRpm,microsoft-centos8-prod-yum,
+SUSE-15Gen2,azcopyAmdRpm,microsoft-sles15-prod-yum,
+Mariner-2.0-x86_64,marinerAmdRpm,cbl-mariner-2.0-prod-Microsoft-x86_64-yum,
+Mariner-2.0-aarch64,marinerArmRpm,cbl-mariner-2.0-prod-Microsoft-aarch64-yum,
+Rocky-8.0,azcopyAmdRpm,microsoft-el8-prod-yum,
+Rocky-9.0,azcopyAmdRpm,microsoft-el9-prod-yum,
```

Comment:
Are we planning to release AzCopy in all the above Linux distros?

Reply:
yes

Reply:
We shall show this to Scott once and see what else we need here